### ISSUE TITLE
panel tab shortcut key

### ISSUE THREAD

**Issue by gintering1000** [dbeaver/dbeaver#15237]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when i switch result panels switch ( calc, value, references, Grouping, Metadata ) by keyboard
Dbeaver only have show/hide ( calc, value, references, Grouping, Metadata ) tab.
i always show that [5 tabs]. 

User Action
01. i execute SQL 
02. switch to result panel with (Switch active panel shortcut key).
03. check rows
04. i can't move to that tabs with keyboard ( 5tabs ) : dbeaver only have show/hide button

**Describe the solution you'd like**
- i think need shortcut key for focus to 5 tabs each other
- and need tab move in panels ( for example : panel tab next, panel tab previous with focus )

be careful
- basically dbeaver have [tab next, tab previous shortcut key]. but that is for file move. it is not for panel tab.

**Describe alternatives you've considered**
i dont know

**Additional context**

**Comment by ShadelessFox** [comment]
Hello @gintering1000,

Is that what you want to achieve? Please see attached video.

https://user-images.githubusercontent.com/35821147/150382984-21b724c1-6695-40f4-8267-a9a4ac9bc09d.mp4

**Comment by gintering1000** [comment]
it is not work work after press shortcut [next tab]

## 1. [ tab next, previous ] is work only 1 time after user press hotkey ( Activate results/panel )
in video, i pressed [tab next] many time.
but it work only 1 time
and i didnt click to value tab like you. i switched by shortcut ( Activate results/panel ) to use that without mouse

![Screen Capture 2022-01-21 at 21 24 59](https://user-images.githubusercontent.com/77244047/150526803-f33eca7f-34a4-4830-8e6a-67adfecfe8b8.gif)
![2022-01-21 21-26-57](https://user-images.githubusercontent.com/77244047/150526962-8f2ad18e-5b77-47f1-99db-382cc9150750.png)


## 2. user can't switch [panel tab next] when they focus editor or console
second problem is [tab next] is work only when i focus to [ result panel ]. i want to change [panel tab next] globally. ( when developer focus on [console or editor], they want to switch panel tab )

**Comment by ShadelessFox** [comment]
Thanks for the clarification.
